### DISCUSSION TITLE
Advanced window customization

### DISCUSSION THREAD

**Discussion by Tiago283** [flet-dev/flet#4232] [Ideas]
Dear programmers, I would like you to add more customization options to the window, especially in the title bar, such as custom buttons, placing the icon directly in the code and adding custom functions(browser tabs, extra buttons, etc.).

**Comment by ap4499** [comment]
Check out: https://flet.dev/docs/reference/types/window/

Frameless or title_bar_hidden might be what you are after - it should allow you to create custom widgets in the titlebar.
